FrontEnd/cavityDisplay.py

```diff
@@ -19,63 +19,59 @@
 from constants import STATUS_SUFFIX, SEVERITY_SUFFIX
 
 
-
 class cavityDisplay(Display):
 
     def ui_filename(self):
-        return 'cavityDisplay.ui'
-        
+        return 'cavityDisplay.ui'   
+    
     def __init__(self, parent = None, args = None):
         super(cavityDisplay, self).__init__(parent=parent,args=args)
-
-        self.ui.linac0.loadWhenShown = False
-        self.ui.linac1.loadWhenShown = False
-        self.ui.linac2.loadWhenShown = False    
-        self.ui.linac3.loadWhenShown = False
-                
-
-        repeaters = [self.ui.linac0,
-                     self.ui.linac1,
-                     self.ui.linac2,
-                     self.ui.linac3]  # type: List[PyDMTemplateRepeater]
         
-        for index, linacTemplateRepeater in enumerate(repeaters):
+        self.ui.L0B.loadWhenShown = False
+        self.ui.L1B.loadWhenShown = False
+        self.ui.L2B.loadWhenShown = False
+        self.ui.L3B.loadWhenShown = False
```

Comment:
I would collapse these into the for loop as `linacEmbeddedDisplay.loadWhenShown = False`